### DISCUSSION TITLE
APM Metrics for Kong

### DISCUSSION THREAD

**Discussion by pmutyala** [Kong/kong#8777] [Help] [answered]
For Kong, we see only Prometheus or DD based regular metrics, is there any 3rd support that can show APM based metrics for Kong ?

**Comment by jschmid1** [comment, marked as answer]
This is the list of [Monitoring & Analytics](https://docs.konghq.com/hub/#analytics-monitoring) plugins that Kong supports.

If you can't find a suitable option in this list, feel free to suggest what's missing.
